pypit/data/settings/settings.keck_deimos

```diff
@@ -370,12 +370,25 @@ dark check condition1 exptime>999999 # Avoids any dark frames
 
 # Make some changes to the arguments and flags
 ## JFH Trying to make slit identification more robust for DEIMOS
-settings trace slits sigdetect 500.0
-settings trace slits number 1
+#settings trace slits sigdetect 500.0
+#settings trace slits number 1
+#settings trace slits tilts params 1,1,1
+#settings trace slits tilts method spca
+#settings trace slits sobel mode constant    # JFH reports this is needed for slits running off the detector
+#settings bias useframe overscan
+#settings pixelflat combine method median
+#settings pixelflat combine reject level [10.0,10.0]
+
+# These are Ryan Cooke's suggested slit detection settings
```

Comment:
probably don't need my name in there! I haven't optimised these settings at all, so it might be better to rethink those choices at some point.